internal/database/online/types.go

```diff
@@ -30,7 +30,7 @@ type PushOpt struct {
 	Entity        *types.Entity
 	EntityKey     string
 	GroupID       int
-	FeatureNames  []string
+	FeatureList   types.FeatureList
```

Comment:
nit: how about `Features`?

Reply:
I think `Features` is more suitable too. but `GetOpt`, `MultiGetOpt`, `ImportOpt` are `FeatureList`, for the sake of unity I wrote `FeatureList` here, maybe I change it in one new PR?